babyagi.py

```diff
@@ -120,6 +185,14 @@ def context_agent(query: str, index: str, n: int):
         print("\033[93m\033[1m"+"\n*****TASK RESULT*****\n"+"\033[0m\033[0m")
         print(result)
 
+        # Store the result in the task_results and completed_tasks table
+        db_cursor.execute(
```

Comment:
code is fine but i think next step is to abstract this out into a task store that uses sqlite (or whatever other pluggable means such as json files)